docs/examples/subroutes.rst

```diff
@@ -0,0 +1,30 @@
+====================
+Example -- Subroutes
+====================
+
+The ``routes`` decorator lets you set up different routes easily, but it can be cumbersome to write many similar routes.
+If you need to write similar routes, the ``subroute`` function can help.
+``subroute`` is a context manager within whose scope all invocations of ``route`` will have a prefix added.
+
+Here is an example app that has routes for ``/branch/lair``, ``/branch/crypt``, and ``/branch/swamp`` all defined in a ``with app.subroute()`` block.
+
+.. code-block:: python
+
+    from klein import Klein
+
+    app = Klein()
+
+    with app.subroute("/branch") as app:
+        @app.route("/lair")
+        def lair(request):
+            return "These stairs lead to the lair of beasts."
```

Comment:
Minor thing -- could you make these functions return bytestrings, so that this doc works unchanged in the impending Python 3 port (https://github.com/twisted/klein/pull/79)?